### PR TITLE
Add g8 templates to the documentation.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
@smarter here is an update for the documentation of the example projects.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ If compiling this example project fails, you probably have a global sbt plugin
 that does not work with dotty, try to disable all plugins in
 `~/.sbt/0.13/plugins` and `~/.sbt/0.13`.
 
+### Tamplate projects
```
Comment (smarter) [inline README.md]: typo: Tamplate -> Template

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ If compiling this example project fails, you probably have a global sbt plugin
...
+The fastest way to have a new project in dotty is using one of the following templates:
+*  [Dotty Hello World](https://github.com/lampepfl/dotty-hello-world.g8)
```
Comment (smarter) [inline README.md]: I would just call the template `dotty.g8` or `dotty-simple.g8`

File: README.md [hunk 3]
```diff
@@ -9,6 +9,11 @@ If compiling this example project fails, you probably have a global sbt plugin
...
+*  [Dotty Hello World (cross compiling with Scala 2)](https://github.com/lampepfl/dotty-hello-world-coss.g8)
```
Comment (smarter) [inline README.md]: typo: `coss` -> `cross`, I would just call this one `dotty-cross.g8`

File: README.md (hunk 2 above)
Comment (nicolasstucki) [inline README.md]: This is a good idea.

**Pull request merged**